roles/ui/files/FWO.UI/Services/ModellingAppServerHandler.cs

```diff
@@ -87,12 +87,25 @@ private async Task<bool> AddAppServerToDb()
         {
             try
             {
+                if (ActAppServer.Ip.TrySplit('-', 1, out string ipEnd))
+                {
+                    ActAppServer.Ip = ActAppServer.Ip.Split('-')[0];
+                    ActAppServer.IpEnd = ipEnd;
+                }
+
+                if (!ActAppServer.Ip.TrySplit('/', 1, out string? subnet))
+                {
+                    ActAppServer.Ip += "/32";
```

Comment:
this needs to be adjusted to not only work for ipv4 (/32) but also for ipv6 (/128)

Reply:
oh right...